src/EUTelProcessorAnalysisPALPIDEfs.cc

```diff
@@ -762,7 +781,7 @@ void EUTelProcessorAnalysisPALPIDEfs::processEvent(LCEvent *evt)
                     nResidualXPixel2by2[chi2Max[i]][index]->Fill(fmod(xposfit,2*xPitch),fmod(yposfit,2*yPitch));
                     nResidualYPixel2by2[chi2Max[i]][index]->Fill(fmod(xposfit,2*xPitch),fmod(yposfit,2*yPitch));
                   }
-                  else if (chi2 < chi2Max[i] && (yposfit < 9.2 || yposfit > 12.7 || xposfit > 27.4 || xposfit < 1.2))
+                  else if (chi2 < chi2Max[i] && (yposfit < 9.2 || yposfit > 12.7 || xposfit > 27.4+1.3 || xposfit < 1.2 - 0.2))
```

Comment:
I think what we see in the analysis results is a convolution of the actual size of the hole with the misalignment and with the size of the hole in the other planes. The values in the code are a conservative estimate which I got by looking at the analysis results, not by measuring the actual size of the hole. I would be fine with updating these values with the actual measured ones, although they still should be variables being read from the steering templates, since different carriers have different hole sizes, so these values could change from measurement to measurement.